src/com/lftechnology/java/training/alina/basic/CalculateGPA.java

```diff
@@ -0,0 +1,29 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Write a program that calculates percentage and GPA of 5 students. User will
+ * enter the marks of each 5 students from the keyboard. Each of the students
+ * has following attributes: name, class, rollnum, and marks obtained in 5
+ * subjects. Calculate the GPA and show the student name and GPA by
+ * position.[higher GPA should display on top]
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class CalculateGPA {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)